.github/workflows/nightly-automated-builds.yaml

```diff
@@ -0,0 +1,26 @@
+name: Nightly Automated Builds
+on:
+  workflow_dispatch:
+  # schedule:
+  #   - cron: "0 6 * * *" #  Every day at 6AM UTC which is 6PM NZDT
```

Comment:
Still getting it to work

Reply:
Have tested on a separate repo and works: https://github.com/GeronimoJohn/gh-action-tags/actions/runs/15840970494